specs/asgi.rst

```diff
@@ -232,9 +232,9 @@ defined by the protocol spec. Examples of a message ``type`` value include
 .. note::
 
   Applications should actively reject any protocol that they do not understand
-  with an Exception (of any type). Failure to do this may result in the server
-  thinking you support a protocol you don't, which can be confusing with
-  the Lifespan protocol, as the server will wait to start until you tell it.
+  with an exception (of any type). Failure to do this may result in the server
```

Comment:
`Exception` is actually the right capitalisation here as it's the name of the root Python type that is expected.

Reply:
I had flipped back and forth on this one; perhaps stylizing as `Exception` vs. Exception is a good compromise there 👍

Reply:
I agree with that.